src/pages/transactions/[id].js

```diff
@@ -41,6 +54,14 @@ export default function TransactionDetails() {
         <h2>Description</h2>
         <div>{transaction.description}</div>
       </div>
+      {mode === "delete" && (
```

Comment:
When testing in the preview deployment, the modal was completely out of sight on my laptop screen, since it pops up at the very bottom of the page. On mobile screens, it was also partially hidden. As this would probably not pass QA, you should either jump to the bottom of the page when the model opens up, or move it somewhere else, where it is always completely visible.

Reply:
see my answer to your last comment